lib/rtp_h264/nal_header.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Membrane.Element.RTP.H264.NALHeader do
+  @moduledoc """
+  Defines a structure representing Network Abstraction Layer Unit Header
+
+  Defined in [RFC6184](https://tools.ietf.org/html/rfc6184#section-5.3)
+
+  ```
+    +---------------+
+    |0|1|2|3|4|5|6|7|
+    +-+-+-+-+-+-+-+-+
+    |F|NRI|  Type   |
+    +---------------+
+  ```
+  """
+
+  @typedoc """
+  A value of 00 indicates that the content of the NAL unit is not
+  used to reconstruct reference pictures for inter picture prediction.
+  Such NAL units can be discarded without risking the integrity
+  of the reference pictures, although these payloads might contain metadata
+  """
+  @type nri :: 0..3
+
+  @typedoc """
+  Specifies the type of RBSP data structure contained in the NAL unit.
+  """
+  @type type :: 1..23
```

Comment:
This can be greater, just as `PaloadTypeDecoder` docs say. Also, I think `PayloadTypeDecoder` could be merged with this module and decode_type would accept `NALHeader` struct